docs/guide/execution/quickrun_execution.rst

```diff
@@ -0,0 +1,41 @@
+.. _userguide_quickrun:
+
+Execution with Quickrun
+=======================
+
+The planning and preparation of a campaign of alchemical simulations using the ``openfe`` package is intended to be
+achievable on a local workstation in a matter of minutes.
+The **execution** of these simulations however requires a large amount of computational power and is intended to be
+distributed onto a HPC environment.
+To do this requires storing and sending the details of the simulation from the local workstation to a HPC environment,
+this can be done via the :func:`.Transformation.dump` function which creates a saved "json" version of the data.
```

Comment:
Should this really be here? It feels like this page should concentrate "I have a JSON, what do I do now", and this should get punted over to the setup section.. maybe we're missing a "writing out an alchemical network" section?

Reply:
I added a link to the relevant cookbook here.  It's duplicated information, but it's also essential for anything else on the page to make sense so it's probably warranted